apps/builder/app/builder/features/style-panel/sections/layout/shared/flex-grid.tsx

```diff
@@ -10,6 +10,248 @@ import type { CreateBatchUpdate } from "../../../shared/use-style-data";
 import { getStyleSource, type StyleInfo } from "../../../shared/style-info";
 import { theme } from "@webstudio-is/design-system";
 
+const Lines = ({
```

Comment:
Flex no longer works to represent itself?

Reply:
<img width="267" alt="Screenshot 2023-04-22 at 20 27 10" src="https://user-images.githubusercontent.com/825702/233798275-a10b145f-170b-4371-a58e-8c878ab3930b.png">

Not straightforward to do this with flex, I decided to export SVG to safe time and make sure we're pixel perfect.

Reply:
Ah, I remember. space-between and space-around were prettified with wrong positions.

Reply:
Probably won't switch to svg after all. Thanks for forcing me to think this through one more time :)